src/components/FacetHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_faceting.scss';
+
+import { useRef, useState } from 'react';
+
+// Components
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+
+type FacetHeaderProps = {
+  key: string;
+  headerText: any;
+  showTooltipIcon: any;
+  tooltipContent: any;
```

Comment:
Use proper `string` type, and let's rename it to `comment` for consistency. It should also be optional.